### DISCUSSION TITLE
Multiple choice polls

### DISCUSSION THREAD

**Discussion by D4ndellion** [element-hq/element-meta#594] [Ideas]
#### What would you like to do?
Answer a multiple choice poll with multiple answers

#### Why would you like to do it?
Often you'd be okay with multiple outcomes. For example for planning around time slots

#### How would you like to achieve it?
By clicking the polls items

### Have you considered any alternatives?

The most often used alternative is to just type out messages and use reactions as polls instead.

### Additional context

The current spec proposal for polls https://github.com/matrix-org/matrix-spec-proposals/pull/3381 has multiple choice in it,
But it seems this was added after some of the implementations were made (IIRC polling existed in RiotX)

This is a pretty low priority thing since the workaround is so simple. But I was surprised when I made a multiple choice poll manually and it couldn't be properly answered. (And didn't see an issue about it) So adding this for tracking purposes.

**Comment by nitn3lav** [comment]
This was already implemented for Element Web. However, the PR https://github.com/matrix-org/matrix-react-sdk/pull/9519 was rejected because it wouldn't be cross platform across all versions of Element. I would be happy to implement this for iOS (both Element and [Element X](https://github.com/element-hq/element-x-ios/issues/2756)) and I might also take a look at Android. Could you please confirm that this would get merged (if my code meets all requirements, etc.)

CC: @dbkr

**Comment by JPustkuchen** [comment]
https://github.com/element-hq/element-meta/issues/2694 is the correct issue to link here. Already has lots of thumbs up!
